app/src/pages/[resourceType]/[id].tsx

```diff
@@ -50,6 +53,10 @@ export default function ResourceIDPage({ jsonData }: InferGetServerSidePropsType
     window.addEventListener('resize', handleResize);
   }, []);
 
+  useEffect(() => {
+    setActiveTab('json');
+  }, [jsonData.id]);
```

Comment:
I'm a bit confused on this `useEffect`. We are subscribing to changes on the value of `jsonData.id`, which is obtained through serverside props, and then setting the active tab to JSON. Is there a need to do this at all? The default value of the tab is already `json`, and the id should never change client side since changing of `jsonData` would happen due to a full re-load of the page server side

Reply:
I definitely believe there is a better way to do this, but the reason I included this` useEffect` is due to an error I was running into whenever a user attempted to navigate to a `dependency artifact` using its link. If the dependency artifact didn't have a dependencies tab of it's own because the "get data requirements" button wasn't clicked yet, the user would be brought to a blank page because it would be on a "dependencies" tab that didn't yet exist. It looked like this for example: 
![Screenshot 2023-07-07 at 12 04 48 PM](https://github.com/projecttacoma/measure-repository/assets/98363677/1ddd0060-0d2f-46da-b373-1614f8101e27)

So I tried to remedy this via the useEffect so whenever the user clicks a dependency artifact's link and gets routed to another page, it would automatically set the JSON tab as active.